tests/ui/simd/issue-89193.rs

```diff
@@ -17,13 +17,13 @@ extern "platform-intrinsic" {
 fn main() {
     let x: [usize; 4] = [10, 11, 12, 13];
     let default = x4(0_usize, 1, 2, 3);
-    let mask = x4(1_i32, 1, 1, 1);
+    let mask = x4(u8::MAX, u8::MAX, u8::MAX, u8::MAX);
```

Comment:
This is actually considered a signed integer type for... some reason, I don't know if it actually matters.

```suggestion
    let all_set = u8::MAX as i8; // aka -1
    let mask = x4(all_set, all_set, all_set, all_set);
```

Reply:
I think `repr(simd)` is okay with either, but we tend to use signed.

Reply:
unsigned doesn't compile (I force-pushed this away): https://github.com/rust-lang/rust/actions/runs/4158968893/jobs/7194581100